cli/server/server_test.go

```diff
@@ -39,14 +40,15 @@ func TestGetConfigFromContext(t *testing.T) {
 
 func TestHandleLoggingParams(t *testing.T) {
 	d := t.TempDir()
-	testLog := path.Join(d, "file.log")
+	testLog := filepath.Join(d, "file.log")
 
 	t.Run("default", func(t *testing.T) {
 		set := flag.NewFlagSet("flagSet", flag.ExitOnError)
 		ctx := cli.NewContext(cli.NewApp(), set, nil)
 		cfg := config.ApplicationConfiguration{
 			LogPath: testLog,
 		}
+		// TODO: see https://github.com/uber-go/zap/issues/621 and https://github.com/uber-go/zap/pull/624
```

Comment:
This needs an issue then.

Reply:
https://github.com/nspcc-dev/neo-go/issues/2269